extensions/rocksdb-repos/RocksDbRepository.cpp

```diff
@@ -20,22 +20,22 @@ using namespace std::literals::chrono_literals;
 
 namespace org::apache::nifi::minifi::core::repository {
 
-void RocksDbRepository::printStats() {
+std::optional<RepositoryMetricsSource::RocksDbStats> RocksDbRepository::getRocksDbStats() const {
+  RocksDbStats stats;
   auto opendb = db_->open();
   if (!opendb) {
-    return;
+    return stats;
```

Comment:
The return type is optional. Didn't you mean to return an empty optional here, instead of a default-constructed RocksDbStats object?

Reply:
I returned the default 0 values when the repository is RocksDB based, but the values are not available at the moment. I returned `std::nullopt` in repositories that are not RocksDB based.